### ISSUE TITLE
Should accept `~` in paths in gpg.edn

### ISSUE THREAD

**Issue by zilti** [adzerk-oss/bootlaces#5]
It would be nice if bootlaces would support gpg certificate paths with the `~`-character in them representing the home dir.

**Comment by micha** [comment]
I don't think we really want to have a special way to parse paths for this one file. I think we're probably better off sticking with the expected java paths across the board, no?
